### ISSUE TITLE
Will this automatically tag & release?

### ISSUE THREAD

**Issue by retorquere** [hahow/probot-conventional-release#8]
Will this automatically tag & release on my project when I have a commit message like `fix: ...`? Can I specify release assets?

**Comment by amowu** [comment]
> Will this automatically tag & release on my project when I have a commit message like fix: ...?

Yes, when you have a pull request merged into the master branch, a changelog is automatically created on the GitHub releases page.

> Can I specify release assets?

Yes, you can edit the automatically created changelog and upload assets.

**Comment by retorquere** [comment]
Will this only work for pull requests? Not when I tag something myself on master?

I realize I can manually edit the upload assets, but can tis bot also automatically publish assets when it creates a release?

**Comment by amowu** [comment]
Hi @retorquere 

> Will this only work for pull requests? Not when I tag something myself on master?

Currently, you can only automate the creation of a changelog through a pull request.

Next time, it will increase the automatic establishment through git push.

If it is a tag, it can only be created manually.

> I realize I can manually edit the upload assets, but can tis bot also automatically publish assets when it creates a release?

There is currently no such plan. I suggest you can try this program.

https://github.com/tcnksm/ghr
https://github.com/aktau/github-release
